### ISSUE TITLE
Better twitter auto-image?

### ISSUE THREAD

**Issue by nathanmwilliams** [townhallproject/townHallProject#321]
I feel like we've discussed this before but there has to be a way to adjust the auto-generated image when we link to the site on twitter. It's a weird zoom in on something (part of the map?) presently.

![twitter image](https://user-images.githubusercontent.com/25805849/37878308-d8f7d7e0-301b-11e8-975a-6d7c54e986e4.png)

**Comment by pdvorchik** [comment]
What browser and OS are you using?  I've tested it on firefox and chrome on windows 10 and it seems to be correct:

https://i.imgur.com/7Ps932X.png?1

**Comment by nathanmwilliams** [comment]
Sorry, should have clarified. This is in Android.

**Comment by pdvorchik** [comment]
Huh It looks okay on android for me as well.  What phone do you have?  We can check resolution

**Comment by nathanmwilliams** [comment]
It's a Motorola G5plus. 1920x1080 resolution. 

If this is really only affecting people with this specific model, certainly becomes a less urgent issue.

**Comment by nathanmwilliams** [comment]
Yeah... I really could have provided more useful info here. The twitter app.

**Comment by meganrm** [comment]
@pdvorchik anymore info here? did we figure out what was causing this?
